### PR TITLE
Registration for 18

### PR DESCRIPTION
**Pull request by shreyassood**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -110,16 +110,13 @@ <h2>Who are the Freetail Hackers?</h2>
 	</div>
 	<object data="assets/us.svg" type="image/svg+xml" width="600px" height="600px"></object>
 </section>
-<section id="mailing">
```
Comment (jaflo) [inline index.html]: Are we removing the entire mailing section?

File: package.json [hunk 2]
```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "http-server": "^0.11.1",
     "node-sass": "^4.7.2",
-    "node-sass-watcher": "^0.5.1"
+    "node-sass-watcher": "^0.5.1",
+    "hoek": "^4.2.1"
```
Comment (jaflo) [inline package.json]: What is this added for? Out of curiosity

File: index.html (hunk 1 above)
Comment (shreyassood) [reply index.html]: I think so?

File: package.json (hunk 2 above)
Comment (shreyassood) [reply package.json]: Not sure how it got there?

**Pull request merged**